app/services/ProductDAO.scala

```diff
@@ -59,30 +67,44 @@ trait ProductDAO {
 
   /** Update statement into the database to update a product given in the product instance using its UUID
    *
-   * @param id
-   * @param product
-   * @tparam T
+   * @param id: String of the UUID given by the user.
+   * @param product: Given product instance to use in query.
+   * @tparam T: What the query will put its data in the format of.
    * @return
    */
   def updateResultById[T](id: String, product: Products): Int = {
-    sql"update Product set name = ${product.name}, quantity = ${product.quantity}, price = ${product.price} where id=${id}::uuid "
+    val status = Try(
+      sql"update Product set name = ${product.name}, quantity = ${product.quantity}, price = ${product.price} where id=$id::uuid "
       .update
       .run
       .transact(xa)
       .unsafeRunSync()
+    )
+
+    status match {
+      case Success(num) => num
+      case Failure(_) => -1
```

Comment:
the return values should be explained on the function comments

Reply:
That makes sense, I'll add it into the comments.